README.md

```diff
@@ -1,23 +1,95 @@
-[![Coverage Status](https://coveralls.io/repos/github/Financial-Times/v1-suggestor/badge.svg)](https://coveralls.io/github/Financial-Times/v1-suggestor)
+[![Circle CI](https://circleci.com/gh/Financial-Times/v1-suggestor.svg?style=shield)](https://circleci.com/gh/Financial-Times/v1-suggestor)[![Go Report Card](https://goreportcard.com/badge/github.com/Financial-Times/v1-suggestor)](https://goreportcard.com/report/github.com/Financial-Times/v1-suggestor) [![Coverage Status](https://coveralls.io/repos/github/Financial-Times/v1-suggestor/badge.svg)](https://coveralls.io/github/Financial-Times/v1-suggestor)
 
 # V1 suggestor
-* Reads V1 metadata from the source kafka queue
-* filters and transforms it to UP standard json representation
-* puts the result on the destination kafka queue
-
-# How to run
-
-```
-export|set SRC_ADDR=http://kafka:8080
-export|set SRC_GROUP=FooGroup
-export|set SRC_TOPIC=FooBarEvents
-export|set SRC_QUEUE=kafka
-export|set SRC_CONCURRENT_PROCESSING=true
-export|set DEST_ADDRESS=http://kafka:8080
-export|set DEST_TOPIC=DestTopic
-export|set DEST_QUEUE=kafka
+Processes metadata about content that comes from QMI system - aka V1 annotations.  
 
+* Reads V1 metadata for an article from the  kafka source topic _NativeCmsMetadataPublicationEvents_
+* Filters and transforms it to UP standard json representation
+* Puts the result onto the kafka destination topic _ConceptSuggestions_
+
+v1-suggestor service communicates with kafka via http-rest-proxy. It polls kafka-rest-proxy for messages and POSTs transfirmed messages to kafka-rest-proxy.  
+This service is deployed in the Delivery clusters.
+## Installation
+
+* `go get -u github.com/Financial-Times/v1-sugestor`
```

Comment:
github.com/Financial-Times/v1-suggestor not github.com/Financial-Times/v1-sugestor